invenio/modules/knowledge/manage.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Perform knowledge operations."""
+
+from __future__ import print_function
+
+import os
+import sys
+
+from invenio.ext.script import Manager
+
+from sqlalchemy.orm.exc import NoResultFound
+
+from .api import add_kb_mapping, get_kb_by_name
+
+manager = Manager(usage=__doc__)
+
+
+@manager.command
+@manager.option('--name', '-w', dest='name')
```

Comment:
Whoops, no you are correct. Bad names. I will rename to for example, `-n` and `-s`